NimbleMedium/Sources/Presentation/Modules/ArticleDetail/ArticleDetailView.swift

```diff
@@ -19,6 +19,7 @@ struct ArticleDetailView: View {
     @State private var isLoadingToastPresented = false
     @State private var isFetchArticleDetailFailed = false
     @State private var isArticleAuthor = false
+    @State private var isEditArticlePresenting = false
```

Comment:
Please make it consistent with other states
```suggestion
    @State private var isEditArticlePresented = false
```